db/schema.rb

```diff
@@ -52,6 +56,12 @@
     t.integer "post_counter", default: 0
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "email"
+    t.string "encrypted_password", default: "", null: false
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.index ["email"], name: "index_users_on_email", unique: true
```

Comment:
I am getting error upon `rails s` to kick the server, and it is affected by the email, also when I want to seed the database, I got this error `ActiveModel::UnknownAttributeError: unknown attribute 'email' for User.` I will encourage you to double the placing of the email.

![image](https://github.com/AJ193/Blog_App_Rails/assets/86200356/66eb9b1e-022f-4782-9146-2f8610fcb27c)